sqlglot/expressions.py

```diff
@@ -5439,6 +5416,34 @@ def maybe_copy(instance, copy=True):
     return instance.copy() if copy and instance else instance
 
 
+def _to_s(node: t.Any, hide_missing: bool = True, level: int = 0) -> str:
+    """Generate a textual representation of an Expression tree"""
+    if isinstance(node, Expression):
+        args = {k: v for k, v in node.args.items() if v is not None or not hide_missing}
+        if node.comments or not hide_missing:
+            args["comments"] = node.comments
+        if node.type or not hide_missing:
+            args["type"] = node.type
```

Comment:
are there any args with called type? there could be one in the future, maybe underscore these or something

Reply:
I added an underscore. Kinda makes it hard to read... but I think that's OK for now since type annotation is a little more advanced.

Reply:
what about renaming `type` to something else? e.g. `dtype` comes to mind, it's used by pandas / numpy as well

Reply:
Actually, `dtype` _is_ an arg, but `type` isn't haha

We can also just append the field to a list of strings, so worst case we get duplicate "type" fields in the representation.

Reply:
Ah I see.. the underscore seems ok for now too, makes sense to have some kind of notation for non-arg attributes